### ISSUE TITLE
How do I use active learning sampling strategies to label data more effectively (using the Community Edition)

### ISSUE THREAD

**Issue by FredrikJoh** [heartexlabs/label-studio#5063]
I am interested in testing to use the active learning capabilities of Label Studio for text classification. I have been able to set up a customized ML model in the backend which can make predictions for my documents, but I do not understand how (and if) I can use the Community Edition version of Label Studio to select the documents to label. In the UI I can only select between random and sequential sampling, but if I understand the documentation correctly it should be possible to customize my active learning loop (even with the Community Edition version). Can I write my own sampling function that can be called upon, and if so, how? If not, can I use uncertainty sampling even if this choice cannot be selected via the UI? I have tried to follow the documentation which says: "To start a project with specific task sampling, see [Set up task sampling for your project](https://docs.humansignal.com/guide/start.html#Set-up-task-sampling-for-your-project)." but that seems to be a broken link.

**Comment by AymanTalkhani** [comment]
@FredrikJoh Not sure if you need images labeled as well, but if you do, have you tried using [InstaLabel](https://www.instalabel.ai/). All you need to do is upload your dataset, select and label a few samples you'd like to finetune the model on in our platform, and hit the 'Train annotator' button. You can then head over to the 'Annotate' tab, click 'Instant Annotation' and have the finetuned model label the rest of your dataset for you. There's even a crowdsourcing option to correct AI predictions for verified ground truth labels. Could be useful for a lot of custom computer vision labeling use cases if you're on the lookout.

**Comment by hogepodge** [comment]
Hi @FredrikJoh, Label Studio Enterprise has a much tighter active learning integration loop. Label Studio Community edition is more basic. You will need to provide your ML model a `fit` method that Label Studio can send data to. After you've annotated at least one piece of data, you can call the `Train Model` method (either from the ML setup pane or through an API call). The annotated data will then be sent to the model for training, and you should be able to update predictions based on that training.
